src/utils/formatValue.ts

```diff
@@ -56,5 +66,5 @@ export const formatStorage: ValueFormatter = (
   _unit,
   { fractionDigits = 2 } = {}
 ) => {
-  return value.toFixed(fractionDigits);
+  return `${unknownTypeFormatter(value, _unit, { fractionDigits })} GB`;
```

Comment:
Do we want to hard code `hours` and `GB` here? Perhaps the component should add the localized labels and leave this util to just format the value itself?

Reply:
From my understanding `ValueFormatter` should do the localized labels.
I might have a better way to format the labels without creating a new function for every new unit.

Reply:
Are you still planning to  localize labels like 'GB' and 'Hours'?

Reply:
Yes sure. I am working on it..